### PR TITLE
[HW1] Eric860730's hw1 first attempt

### PR DESCRIPTION
**Pull request by Eric860730**
---
A good PR should include the following parts:

- [x] A clear title
- [x] A meaningful message for PR, as well as its commits
- [x] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [x] excluding any irrelevant files, such as binaries, text files, or dot files
- [x] passing tests/CI

### CODE REVIEW

**Comment by yungyuc** [comment]
@q40603 action hasn't run on this but I do not know why.  Could you take a look when you have time?

**Comment by Eric860730** [comment]
@yungyuc hello teacher, the github action seems to be triggered after i push again.

**Comment by q40603** [comment]
I just approved the action. Don't know why it didn't trigger before.

**Comment by Eric860730** [comment]
@q40603 thank you.

File: hw0/Eric860730/countline.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```
Comment (yungyuc) [inline hw0/Eric860730/countline.py:1]: You do not need to turn in `hw0`.  And please remove unneeded file like this from your `hw1` submission.

**Review by yungyuc — CHANGES_REQUESTED** [review]

**Review by yungyuc — APPROVED** [review]

**Pull request merged**